internal/backends/winit/renderer/femtovg/fonts.rs

```diff
@@ -231,8 +231,13 @@ impl Default for FontCache {
             };
             font_db.set_sans_serif_family(default_sans_serif_family);
         }
-        let available_families =
-            font_db.faces().iter().map(|face_info| face_info.family.as_str().into()).collect();
+        let available_families = font_db
+            .faces()
+            .iter()
+            .flat_map(|face_info| {
+                face_info.families.first().map(|(family_name, _)| family_name.as_str().into())
```

Comment:
I don't know what `families`  is, but shouldn't that be something like `iter()` instead of `first`?

Reply:
`families` is a vector of all the names of the font in different languages. The first entry of the vector is always the English name for the font, unless it is missing. So at this point I'm certain that we only want the first entry of the vector.